websites/management/commands/markdown_cleaning/parsing_utils.py

```diff
@@ -127,7 +127,7 @@ class ShortcodeParam:
     name: Union[str, None] = None
 
     param_regex: ClassVar[re.Pattern] = re.compile(
-        r"^((?P<name>[0-9a-zA-Z_]+)=)?(?P<value>.*)$"
+        r"^((?P<name>[0-9a-zA-Z_\-]+)=)?(?P<value>.*)$"
```

Comment:
In particular, the `data-ngdescrip` shortcode param in `< image-gallery-item >` necessitated this change.